23cfree/schema-priv-setup/setup.md

```diff
@@ -20,105 +20,107 @@ In order to do this workshop you need
 * Insert data into the inventory\_no\_reservations table with two rows: Product A and Product B.
 * Insert data into the inventory\_reservations table with two rows: Product C and Product D.
 
-## 
-## Task 1: Create Users 
+##

```

Comment:
@bhenndricks - Could you remove the two ## before Task 1 in all three markdown files?

Reply:
Hi Anoosha,

It's odd but I have gotten on a call with him to look at his files. For some reason if he doesn't have that two ## before Task 1, task 1 will not be underneath the fold. I didn't know how else to resolve it. Perhaps you can help guide him if there's a better way. 

Here's the lab as is: https://oracle-livelabs.github.io/database/23cfree/workshops/sandbox-schemapriv-lockfree/index.html?lab=setup

Reply:
Thanks Anoosha! 
@bhenndricks please create a new commit for this before we can approve your PR